pypeit/core/extract.py

```diff
@@ -1626,9 +1639,10 @@ def remap_orders(xinit, spec_min_max, inverse=False):
                                                              bounds_error=False, fill_value='extrapolate')(spec_vec_norm)
     return xinit_remap
 
-# TODO Can we purge OBJID in this routine in place of ech_objid everywhere?
-def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, inmask=None, spec_min_max=None,
-                fof_link=1.5, order_vec=None, plate_scale=0.2, ir_redux=False,
+
+def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, order_vec, maskslits,
```

Comment:
`order_vec` is always used and is therefore required.

Reply:
The code works just fine with order_vec generated as order indices which is how it was written before. Just because order_vec is aways used currently does not mean that all usages will need it. Please change it back.

Reply:
This is fine for now. Just add a lines to the docs for order vector, that the code writes order number to to the sobjs, but if you don't have those you can use np.arange(orders).

Reply:
done.